README.md

```diff
@@ -6,7 +6,7 @@ The JMX Exporter is a collector to capture JMX MBean values.
 
 # Documentation
 
-Documentation can be found at [http://prometheus.github.io/jmx_exporter/](http://prometheus.github.io/jmx_exporter)
+Documentation can be found at [JMX Exporter on prometheus.github.io](https://prometheus.github.io/jmx_exporter)
```

Comment:
While the change to https:// for URL is valid...

```
Documentation can be found at [JMX Exporter on prometheus.github.io]
```

... I'm not sure this change is required.

Reply:
Screen readers don't really enjoy reading urls to users and users don't really enjoy hearing them try. There's a reason that we don't usually force users to look at raw urls. That said. I'd rather drop the change than debate it.